src/protocol/thread/pause.ts

```diff
@@ -451,9 +451,18 @@ export class Pause {
       return this.loadMouseTargetsWaiter.promise;
     }
     this.loadMouseTargetsWaiter = defer();
-    const { elements } = await this.sendMessage(client.DOM.getAllBoundingClientRects, {});
-    this.mouseTargets = elements;
-    this.loadMouseTargetsWaiter.resolve();
+    let rv;
+    try {
+      rv = await this.sendMessage(client.DOM.getAllBoundingClientRects, {});
+      this.mouseTargets = rv.elements;
+    } catch (e) {
+      console.error("DOM.getAllBoundingClientRects failed", e);
+    } finally {
+      this.mouseTargets = [];
```

Comment:
setting `mouseTargets` to an empty array prevents `getMouseTarget` and other consumers from failing. Another option would be to set it to `null` and add a guard in the consumers

Reply:
This code is broken, this will overwrite the mouseTargets assignment that happens in the successful case.  This mouseTargets assignment belongs in the catch block.

Also note that inspector-02 (which tests the node picker) is failing in CI:

Failure inspector-02.js https://app.replay.io/recording/233304ca-6cff-4db7-8b1e-bd7ae55328ea